app/src/androidTest/java/RecyclerViewTestExtension.kt

```diff
@@ -0,0 +1,16 @@
+import androidx.annotation.IdRes
+import androidx.recyclerview.widget.RecyclerView
+import androidx.test.espresso.Espresso
+import androidx.test.espresso.ViewAction
+import androidx.test.espresso.contrib.RecyclerViewActions
+import androidx.test.espresso.matcher.ViewMatchers
+
+fun actionOnRecyclerViewItem(@IdRes viewId: Int, viewItem: Int, action: ViewAction) {
```

Comment:
It is some solution, but it isn't extension function. 
Read this: https://kotlinlang.org/docs/extensions.html